addons/sale/views/mail_activity_plan_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="mail_activity_plan_action_sale_order" model="ir.actions.act_window">
```

Comment:
Unnecessary `data` node, can be remove and `record` node indentation can be reduced ;)